source/basic.tex

```diff
@@ -5856,11 +5856,12 @@
 
 \pnum
 It is \impldef{whether the thread that executes \tcode{main} and the threads created
-by \tcode{std::thread} provide concurrent forward progress guarantees} whether the
+by \tcode{std::thread} or \tcode{std::jthread} provide concurrent forward progress guarantees} whether the
 implementation-created thread of execution that executes
 \tcode{main}\iref{basic.start.main} and the threads of execution created by
-\tcode{std::thread}\iref{thread.thread.class} provide concurrent forward progress
-guarantees.
+\tcode{std::thread}\iref{thread.thread.class}
+or \tcode{std::jthread}\iref{thread.jthread.class}
```

Comment:
Should this be "**and** `std::jthread`", not "or `std::jthread`"?

The rule applies to threads created by `std::thread` and threads created by `std::jthread`, not one or the other.

If "and" is used, it should be changed in both places in `support.tex` and also `algorithms.tex` which mirrors this wording and cross-references it.

Reply:
@jwakely As I read it, it is saying exactly what you want, that is, the rule applies to any thread created by either std::thread or std::jthread.  If this were changed to "and", it would only apply to threads created by both std::thread and std::jthread (i.e. the intersection of the threads created by each), which I presume can't happen.  Right?  If this is still wrong, please file an issue with LWG.  Thanks!